tests/sentry/utils/appleconnect/test_itunes_connect.py

```diff
@@ -0,0 +1,193 @@
+"""Tests for iTunes integration.
+
+These tests talk directly to Apple's iTunes and are therefore unsuitable for running in CI.
+Firstly you need to create a file named ``credentials.json`` in this directory::
+
+   {
+     "username": "you-itunes-email@example.com",
+     "password": "your itunes password"
+   }
+
+You account needs to have access to the "GetSentry LLC" organisation for all tests to run.
+Tests which require this are marked with ``getsentryllc`` so you can disable these tests
+using `pytest -m 'not getsentryllc'`.
+
+They also need interactive login which you need to do before running the tests::
+
+   $ sentry execfile tests/sentry/utils/appleconnect/test_itunes_connect.py
+   two-factor code: xxxxxx
+
+To test the SMS authentication flow you need to invoke:
+
+   $ sentry execfile tests/sentry/utils/appleconnect/test_itunes_connect.py sms
+   sms code: xxxxxx
+
+And finally, once logged in you can run the tests with ``--itunes``.
+
+   $ pytest --itunes tests/sentry/utils/appleconnect/test_itunes_connect.py
+"""
+
+import dataclasses
+import pathlib
+import sys
+
+import pytest
+
+from sentry.utils import json
+from sentry.utils.appleconnect import itunes_connect
+
+pytestmark = pytest.mark.itunes
+
+
+@pytest.fixture  # type: ignore
+def here() -> pathlib.Path:
+    return pathlib.Path(__file__).parent
+
+
+@pytest.fixture  # type: ignore
+def session_cookie(here: pathlib.Path) -> str:
```

Comment:
I believe reading the session cookie from disk can be done per-session, probably outside of the fixture as well

then you don't do it for every test anew

Reply:
once per session, yes.  but not outside of a fixture, that's not code you want to run at import time.

Reply:
turns out you can't do that as it's used by the client which is function-scoped